### PR TITLE
UI/design feedback

### PR DESCRIPTION
**Pull request by jmzwar**
Various UI updates. See here https://www.figma.com/file/I6quJpgHEeX5CPjaC3qxOa/Q%26A-StoryBook?node-id=31%3A652

### CODE REVIEW

File: v2/components/Burn/Burn.tsx [hunk 1]
```diff
@@ -112,7 +115,7 @@ export const BurnUi = ({
           </Text>
           <Tooltip label="Soonthetix" hasArrow>
             <Flex alignItems="center">
-              <InfoIcon width="16px" height="16px" />
+              <InfoIcon width="12px" height="12px" />
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply v2/components/Burn/Burn.tsx:118]: I'll rebase off joeys

[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/Burn/Burn.tsx:124]: Yea I think the info icon is 10px in most places. And thats the default of the info icon

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 2]
```diff
@@ -120,19 +122,19 @@ const MaintainActionCard: React.FC<
       {isStaking && (
         <Badge
           color={variant}
-          bg="blackAlpha.600"
+          bg="#47FAC240"
```
Comment (bachstatter) [inline v2/components/MainActionCards/MainActionCards.tsx:125]: No chakra color for this?

File: v2/components/Burn/Burn.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmzwar) [reply v2/components/Burn/Burn.tsx:124]: I'm going off this feedback to make 12px
https://www.figma.com/file/I6quJpgHEeX5CPjaC3qxOa/Q%26A-StoryBook?node-id=31%3A652

File: v2/components/MainActionCards/MainActionCards.tsx (hunk 2 above)
Comment (jmzwar) [reply v2/components/MainActionCards/MainActionCards.tsx:125]: There is but the chakra theme colors don't work with opacity modifiers - that is why I have it like this (this is #47FAC2 40%). There could be a util somewhere from chakra but couldn't find

File: v2/components/Welcome/Welcome.tsx [hunk 3]
```diff
@@ -18,14 +18,7 @@ export const Welcome = ({ ...props }: BoxProps) => {
...
         </Text>
-        <Button
-          bg="transparent"
-          bgImage="none"
-          color="cyan.400"
-          borderWidth={1}
-          borderColor="cyan.400"
-          onClick={() => console.log('To introduction')}
-        >
+        <Button variant="outline" onClick={() => console.log('To introduction')}>
```
Comment (bachstatter) [inline v2/components/Welcome/Welcome.tsx:21]: Add a todo here? Do we need this for first release?

**Review by bachstatter — APPROVED** [review]
Some minor comments. But mostly looks good

File: v2/components/Welcome/Welcome.tsx (hunk 3 above)
Comment (jmzwar) [reply v2/components/Welcome/Welcome.tsx:21]: Yes

File: v2/components/Burn/Burn.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/Burn/Burn.tsx:127]: Yea ok fair enough. I would prefer to change the default values for the Info icon

**Pull request merged**